packages/ui/components/form/checkbox/Checkbox.tsx

```diff
@@ -5,7 +5,7 @@ import classNames from "@calcom/lib/classNames";
 
 type Props = InputHTMLAttributes<HTMLInputElement> & {
   label?: React.ReactNode;
-  description: string;
+  description: string | React.ReactNode;
```

Comment:
I added the `React.ReactNode` type to `description` because I wanted to pass a `<Trans>` component. @sean-brydon